scripts/hero1-battery-manager-laptop.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
@PetervDooren  This services file is wrong. replace by 2 services files. one for hero2 and one for hero2-demo